samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -278,6 +271,123 @@ Testing with nRF Connect for Desktop
 #. Swap roles.
    Set the application to the RX mode and the connected development kit to the TX mode.
 
+Direct Test Mode terminal connection
+------------------------------------
+
+To run Direct Test Mode (DTM) to send commands and receive responses, you can connect to your development kit with RealTerm in Windows or Minicom in Linux.
+
+The Bluetooth Low Energy DTM UART interface standard specifies the following configuration:
+
+* Eight data bits
+* No parity
+* One stop bit
+* No hardware flow control
+* A selection of bit rates from 9600 to 1000000, one of which must be supported by the DUT.
+  It might be possible to run other bit rates by experimenting with parameters.
+
+.. note::
+   The default bit rate of the DTM UART driver is 19200 bps, which is supported by most certified testers.
+
+You have to send all commands as two-byte Hex numbers. The responses must have the same format.
+
+Connect with RealTerm (Windows)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The RealTerm terminal program offers a graphical interface for setting up your connection.
+
+.. figure:: /images/realterm.png
+   :alt: RealTerm start window
+
+   The RealTerm start window
+
+To test DTM with RealTerm, complete the following steps:
+1. On the :guilabel:`Display` tab, set :guilabel:`Display As` to :guilabel:`Hex[space]`.
+
+   .. figure:: /images/realterm_hex_display.png
+      :alt: Set the RealTerm display format
+
+2. Open the :guilabel:`Port` tab and configure the serial port parameters:
+
+   a. Set the :guilabel:`Baud` to 19200 **(1)**
+   b. Select your Jlink serial port from the :guilabel:`Port` list **(2)**
+   c. Set the port status to "Open" **(3)**
+
+   .. figure:: /images/real_term_serial_port.png
+      :alt: RealTerm serial port settings
+
+3. Open the :guilabel:`Send` tab:
+
+   a. Write the command as a Hex number in the field **(1)**.
+      For example, write **0x00 0x00** to send a **Reset** command.
+   b. Click the :guilabel:`Send Numbers` button **(2)** to send the command.
+   c. Observe the response in the DTM in area **(3)**.
+      The response is encoded as Hex numbers.
+
+   .. figure:: /images/realterm_commands.png
+      :alt: RealTerm commands sending
+
+Connect with Minicom (Linux)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Minicom is a serial communication program that connects to the DTM device.
+
+On the Linux operating system, install a Minicom terminal.
+On Ubuntu, run:
+
+.. code-block::
+
+   sudo apt-get install minicom
+
+1. Run the Minicom terminal:
+
+   .. code-block::
+
+      sudo minicom -D <DTM serial port> -s
```

Comment:
We shouldn't use <> for variables.
Use italics instead, and .. parsed-literal:: to make the italics display in a code block.
See for example the "Configuring wpantund" section here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/_sources/ug_thread_tools.rst.txt

Reply:
Sure. Done